pkg/infrastructure/nutanix/clusterapi/clusterapi.go

```diff
@@ -0,0 +1,212 @@
+package clusterapi
+
+import (
+	"context"
+	"fmt"
+
+	nutanixclientv3 "github.com/nutanix-cloud-native/prism-go-client/v3"
+	"github.com/sirupsen/logrus"
+	"k8s.io/utils/ptr"
+
+	infracapi "github.com/openshift/installer/pkg/infrastructure/clusterapi"
+	nutanixtypes "github.com/openshift/installer/pkg/types/nutanix"
+)
+
+// Provider is the Nutanix implementation of the clusterapi InfraProvider.
+type Provider struct {
+	infracapi.InfraProvider
```

Comment:
You don't need to embed this interface (my original example did this mistakenly).

Reply:
Fixed